### PR TITLE
docs: Revised the HTML tags

### PR DESCRIPTION
**Pull request by aanchal88**

### CODE REVIEW

**Review by crisbeto — COMMENTED** [review]
@aanchal88 these changes are appreciated but they also add a lot of noise. Can you combine them all in one PR?

**Comment by aanchal88** [comment]
Hii @crisbeto, I tried to cover them all in this PR. These are only the tags that were remaining.

File: src/material/schematics/ng-generate/mdc-migration/rules/tree-traversal.spec.ts [hunk 1]
```diff
@@ -160,16 +160,16 @@ describe('#visitElements', () => {
     });
 
     it('should handle multiple unnested', async () => {
-      runAddAttributeTest('<a></a><b></b>', '<a add="val"></a><b add="val"></b>');
+      runAddAttributeTest('<a></a><strong></strong>', '<a add="val"></a><b add="val"></b>');
```
Comment (crisbeto) [inline src/material/schematics/ng-generate/mdc-migration/rules/tree-traversal.spec.ts:163]: It looks like the tests here need to be updated as well.

Comment (aanchal88) [reply src/material/schematics/ng-generate/mdc-migration/rules/tree-traversal.spec.ts:163]: For this one 
'<a add="val"></a><b add="val"></b>' ?

Comment (aanchal88) [reply src/material/schematics/ng-generate/mdc-migration/rules/tree-traversal.spec.ts:163]: For this one `'<a add="val"></a><b add="val"></b>'` ?

**Comment by aanchal88** [comment]
Hii @crisbeto , updated the changes please have a look.
Thanks

**Comment by aanchal88** [comment]
Hii @crisbeto , please review the PR.
Thanks

**Comment by aanchal88** [comment]
Hii @crisbeto , fixed the tcs.

**Comment by mmalerba** [comment]
Thanks for the fix!

**Review by mmalerba — APPROVED** [review]

**Pull request merged**